db/services/movie.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional, List
+
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: Optional[List[int]] = None,
+        actors_ids: Optional[List[int]] = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids).distinct()
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: Optional[List[int]] = None,
+        actors_ids: Optional[List[int]] = None,
+) -> Movie:
+    movie = Movie.objects.create(
+        title=movie_title,
+        description=movie_description,
+    )
+    if genres_ids:
+        movie.genres.add(*genres_ids)
```

Comment:
As per checklist:
`Use the .set() method to add a couple of many-to-many relationships (imagine skills is ManyToManyField).`